### PR TITLE
change format in messages and outputs

### PR DESCRIPTION
**Pull request by jzuniga-amd**

### CODE REVIEW

File: clients/benchmarks/client.cpp
Comment (tfalders) [inline clients/benchmarks/client.cpp:33]: Spelling error on getrf

File: clients/rocblascommon/program_options.hpp
Comment (tfalders) [inline clients/rocblascommon/program_options.hpp:388]: Correction: equal to 30

**Review by tfalders — APPROVED** [review]
LGTM! Let me know once you want it merged in.

**Comment by jzuniga-amd** [comment]
Ready to merge if you agree...

**Pull request merged**
